src/libraries/System.Net.Http/src/System/Net/Http/Headers/EntityTagHeaderValue.cs

```diff
@@ -23,19 +23,7 @@ public bool IsWeak
             get { return _isWeak; }
         }
 
-        public static EntityTagHeaderValue Any
-        {
-            get
-            {
-                if (s_any == null)
-                {
-                    s_any = new EntityTagHeaderValue();
-                    s_any._tag = "*";
-                    s_any._isWeak = false;
-                }
-                return s_any;
-            }
-        }
+        public static EntityTagHeaderValue Any => s_any ??= new EntityTagHeaderValue() { _tag = "*" };
```

Comment:
Move `_tag = "*"` into the private constructor, and delete `= null!` initialization of the `_tag`?

Reply:
Makes sense.